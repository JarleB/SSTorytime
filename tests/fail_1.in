
-missing item test # fail

 character (english for pinyin) hànzì
     "      (english for hanzi)  汉字

 apple (pinyin) píngguǒ
       (hanzi) 苹果 

  shuǐ (pinyin for english) water
       (pinyin for hanzi) 水

  X (stands for) person
  Y (stands for) thing
