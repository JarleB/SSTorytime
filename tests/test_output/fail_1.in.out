
5:[31;1;1mN4L fail_1.in No such arrow has been declared in the configuration: pinyin at line 5 [0m
